srcpkgs/ngspice/template

```diff
@@ -1,28 +1,62 @@
 # Template file for 'ngspice'
 pkgname=ngspice
-version=33
+version=34
 revision=1
 build_style=gnu-configure
-configure_args="--with-readline=yes --with-ngshared --enable-xspice --enable-cider"
+configure_args="--with-ngshared --enable-xspice --enable-cider"
 hostmakedepends="bison byacc"
 makedepends="readline-devel libX11-devel libXaw-devel"
 short_desc="Mixed Mode Mixed Level Circuit Simulator based on Spice3F5"
 maintainer="Toyam Cox <Vaelatern@voidlinux.org>"
 license="BSD-3-Clause"
 homepage="http://ngspice.sourceforge.net"
 distfiles="${SOURCEFORGE_SITE}/ngspice/ng-spice-rework/${version}/${pkgname}-${version}.tar.gz"
-checksum=b99db66cc1c57c44e9af1ef6ccb1dcbc8ae1df3e35acf570af578f606f8541f1
+checksum=2263fffc6694754972af7072ef01cfe62ac790800dad651bc290bfcae79bd7b5
+
+# Configure can only handle building ngspice as a library or as a binary, but not both,
+#  so essentially we have to build ngspice twice
+pre_configure() {
+	install -d .build_bins
+	cp -fpr * .build_bins/
+}
```

Comment:
We can avoid some of this if it supports out of tree builds. So instead you'd do `../configure ...`.

Could indicate it with `configure_script`

Reply:
I am pretty unfamiliar with out of tree builds, but does look like it supports it as long as the subdir is configured first.  I reworked the template a bit but this seems to be a good solution.